### PR TITLE
Update screenshots and rearrange content

### PR DESCRIPTION
**Pull request by abhi18av**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -105,7 +105,7 @@ nav:
       - 'Processes': 'monitoring/processes.md'
       - 'Aggregate stats & load': 'monitoring/aggregate_stats.md'
       - 'Tasks & metrics': 'monitoring/tasks.md'
-      - 'Sharing pipelines': 'monitoring/sharing.md'
+      # - 'Sharing pipelines': 'monitoring/sharing.md'
```
Comment (abhi18av) [inline mkdocs.yml:108]: Disabling a feature through docs 😉

File: docs/launch/notifications.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ description: 'Guide to set up notification for Nextflow pipelines using Tower.'
 
 ## Email Notifications
 
-You can receive email notifications at the completion of a  workflow execution.
+You can receive email notifications at the completion of failure of a workflow execution.
```
Comment (tcrespog) [inline docs/launch/notifications.md:9]: `of failure of a` -> `or a failure of a`

File: docs/launch/notifications.md [hunk 3]
```diff
@@ -6,9 +6,9 @@ description: 'Guide to set up notification for Nextflow pipelines using Tower.'
...
 
-Select your profile page on the top right of the window and select the **Send notification email on workflow completion** toggle option at the bottom of the page.
+Naviate to your [profile page](https://tower.nf/profile) using dropdown on your avatar (top-right) and then select the **Send notification email on workflow completion** toggle option at the bottom of the profile settings page.
```
Comment (tcrespog) [inline docs/launch/notifications.md:11]: `Naviate` -> `Navigate`

File: docs/monitoring/overview.md [hunk 4]
```diff
@@ -4,24 +4,23 @@ headline: 'Monitoring Pipelines'
...
+The colors signify the completion status:
 
   - **Blue** are running.
   - **Green** are successfully executed.
   - **Yellow** are successfully executed where some tasks failed.
   - **Red** are jobs where at least one task fully failed.
   - **Grey** are jobs that where forced to stop during execution.
 
-  Selecting a run on the left panel will display the job execution details.
+Selecting any particular run from the panel will display that runs execution details.
```
Comment (tcrespog) [inline docs/monitoring/overview.md:23]: `that runs` -> `that run`

**Review by tcrespog — COMMENTED** [review]
Leaving some comments of what are (I believe) typos. Please, ignore those that might not actually apply.

File: docs/launch/notifications.md (hunk 2 above)
Comment (abhi18av) [inline docs/launch/notifications.md:9]:
```suggestion
You can receive email notifications at the completion or a failure of a workflow execution.
```

File: docs/launch/notifications.md (hunk 3 above)
Comment (abhi18av) [inline docs/launch/notifications.md:11]:
```suggestion
Navigate to your [profile page](https://tower.nf/profile) using dropdown on your avatar (top-right) and then select the **Send notification email on workflow completion** toggle option at the bottom of the profile settings page.
```

File: docs/monitoring/overview.md (hunk 4 above)
Comment (abhi18av) [reply docs/monitoring/overview.md:23]:
```suggestion
Selecting any particular run from the panel will display that run execution details.
```

I think I'll take this one too as Grammarly seems happy about this 😉

**Review by abhi18av — COMMENTED** [review]
Accomodate review comments from Tomás - thanks mate!

**Comment by alvaromartmart** [comment]
At some point we could automatise the documentation screenshots with cypress 👀 .

**Comment by abhi18av** [comment]
Merging to trigger the build for staging website.

**Pull request merged**